src/features/DebugSession.ts

```diff
@@ -648,9 +650,10 @@ export class PickPSHostProcessFeature extends LanguageClientConsumer {
             matchOnDescription: true,
             matchOnDetail: true,
         };
+
         const item = await window.showQuickPick(items, options);
 
-        return item ? item.pid : undefined;
```

Comment:
This was causing the bug because `pid` was a number, not a string and something made it no longer "just work" and instead be invalid and crash.

Reply:
The smaller fix I first tested was just to `ToString()` it in the `item` above...but honestly we should just fix this the right way.

Reply:
Makes sense